app/models/ability.rb

```diff
@@ -72,4 +74,8 @@ def can_manage_links
   def can_manage_self
     can %i[show manage_password], User, id: user.id
   end
+
+  def can_show_defendant
```

Comment:
[see](https://github.com/ministryofjustice/laa-court-data-ui/pull/231/files#r464490267)

Reply:
👍 Added can %i[show], CourtDataAdaptor::Resource::Defendant to can_search